minion/server/routes.py

```diff
@@ -22,6 +23,7 @@ def get_jobs_page(page=1, page_size=10):
     return Job.objects.paginate(page=page, per_page=page_size)
 
 
+@ensure_mongo
 @api.route("/", methods=["GET"])
```

Comment:
The `ensure_mongo` decorator is kind of inappropriate here because this is the route to return the page's HTML.

We should either do another route to fetch the mongodb connection status and render it on the page or we render it to the page with a `render_template` kwarg (good enough for a first solution). 

For the first option:
The route would be called in an interval with Ajax - or we could use websockets to push status updates.